src/AutoNumeric.js

```diff
@@ -8713,8 +8713,7 @@ To solve that, you'd need to either set \`decimalPlacesRawValue\` to \`null\`, o
     /**
      * Process deletion of characters when the minus sign is to the right of the numeric characters.
      *
-     * @param {string} left The part on the left of the caret or selection
-     * @param {string} right The part on the right of the caret or selection
+     * @param {[string, string]} [left, right] The part on the left and on the right of the caret or selection
```

Comment:
The CI tests fails while choking on that JSDoc syntax.
The JSDoc is indeed wrong, but the change must be accepted by the linter

Reply:
Fixed it by documenting the param and return type as string[] with more info about the array elements in the description.